pkg/api/meta.go

```diff
@@ -89,3 +90,32 @@ func (meta *ObjectMeta) GetLabels() map[string]string                 { return m
 func (meta *ObjectMeta) SetLabels(labels map[string]string)           { meta.Labels = labels }
 func (meta *ObjectMeta) GetAnnotations() map[string]string            { return meta.Annotations }
 func (meta *ObjectMeta) SetAnnotations(annotations map[string]string) { meta.Annotations = annotations }
+
+func setMetaOwnerReference(dest *metatypes.OwnerReference, reference *OwnerReference) {
```

Comment:
functions like this are best if named "xToY" or "yFromX".

but actually it's only used in one place, so I'd just inline it. Same for the reverse function.